### PR TITLE
TW-1108 Remove prerequisites and next steps from right sidebar

### PR DESCRIPTION
**Pull request by claireriley**
🙃

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM! I tested several pages and confirmed each deleted section is followed by ---, -type:, or warning:

File: src/pages/docs/collections/using-newman-cli/installing-running-newman.md [hunk 1]
```diff
@@ -23,11 +18,7 @@ contextual_links:
   - type: link
     name: "Newman: run and test your collections from the command line"
     url: "https://blog.postman.com/newman-run-and-test-your-collections-from-the-command-line/"
-  - type: section
-    name: "Next steps"
-  - type: link
-    name: "Postman API documentation"
-    url: "/docs/developer/postman-api/intro-api/"
+warning: false
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/installing-running-newman.md:21]: It's my understanding that this is a deprecated front matter key. Should we hold off from adding this?

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good! I add a question about the `warning` keys we added in this PR.

File: src/pages/docs/collections/using-newman-cli/installing-running-newman.md (hunk 1 above)
Comment (claireriley) [reply src/pages/docs/collections/using-newman-cli/installing-running-newman.md]: Thanks for spotting this! I don't remember touching those keys at all so no idea how these random additions snuck in..the only thing I can think of is I was resolving a bunch of merge conflicts. Anyways! I think I've removed all the "green" ones.

**Pull request closed without merging**